src/gui/mainwindow.cpp

```diff
@@ -27,6 +27,10 @@
 
 namespace gpui {
 
+#define MAIN_WINDOW_GEOMETRY "mainwindow/geometry"
```

Comment:
- [ ] Please use const QString instead of #define, in this project defines provide reflection capabilities and code generation for plugins, so using them for constants will produce convoluted code.
- [ ] Please also provide conventional commit message for this commit e.g.: "feat, ui: add ability to save application geometry state"